draft-ietf-quic-http.md

```diff
@@ -785,10 +785,9 @@ amount of data a server may commit to the pushed stream.
 
 If a promised server push is not needed by the client, the client SHOULD send a
 CANCEL_PUSH frame. If the push stream is already open or opens after sending the
-CANCEL_PUSH frame, a QUIC STOP_SENDING frame with an appropriate error code can
```

Comment:
To me the subject and the verb being omitted in this sentence was a bit surprising, but that might be due to me not being a native speaker.

Reply:
It is an error, thanks for picking it up. I would convert this to

"a QUIC STOP_SENDING frame with an error code of HTTP_REQUEST_CANCELLED can be used."

Alternatively, elsewhere we talk about "triggering QUIC STOP_SENDING" which is under review as part of @MikeBishop #2805